arviz/plots/violintraceplot.py

```diff
@@ -76,11 +80,10 @@ def violintraceplot(trace, varnames=None, quartiles=True, credible_interval=0.94
         ax[axind].plot([0, 0], hpd_intervals, lw=linewidth, color='k', solid_capstyle='round')
         ax[axind].plot(0, per[-1], 'wo', ms=linewidth*1.5)
 
-        ax[axind].set_xlabel(names[axind], fontsize=textsize)
+        ax[axind].set_xlabel(var_name, fontsize=textsize)
```

Comment:
I'd use `for axind, (var_name, selection, x) in enumerate(plotters):` above, and then use `make_label(var_name, selection)` here.